examples/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project(ArrayFire-Examples
   VERSION 3.5.0
   LANGUAGES CXX)
 
+if (NOT ASSETS_DIR)
+    option(ASSETS_DIR "Please provide the path to assets for ArrayFire examples")
```

Comment:
Option creates a boolean variable. You want to create a variable using the set command that is cached if the examples are not built as part of the ArrayFire library.

Reply:
Fixing it.